pkg/config/config_template.yaml

```diff
@@ -289,6 +289,31 @@ api_key:
 #
 # forwarder_requeue_buffer_size: 100
 
+## @param forwarder_backoff_factor - int - optional - default: 2
+## @env DD_FORWARDER_BACKOFF_FACTOR - integer - optional - default: 2
+## Defines the number to control the overlap between consecutive retry interval ranges.
+# forwarder_backoff_factor: 2
+
```

Comment:
I think this option should remain undocumented, since it should never be useful to end users